applications/protocols_serialization/server/snippets/log_rpc/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ / {
+	sram@2003FC00 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x2002EC00 DT_SIZE_K(1)>;
+		zephyr,memory-region = "RetainedMem";
+		status = "okay";
+
+		retainedmem {
+			compatible = "zephyr,retained-ram";
+			status = "okay";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			/* This creates a 1kB partition for crash log */
+			crash_log: retention@0 {
+				compatible = "zephyr,retention";
+				status = "okay";
+				reg = <0x0 DT_SIZE_K(1)>;
+
+				/* This is the prefix which must appear at the front of the data */
+				prefix = [ 01 00 00 00  ];
+			};
+		};
+	};
+};
+
+/* Reduce SRAM_CPUAPP usage by 1KB to account for non-init area */
+&cpuapp_sram {
+	reg = <0x20000000 DT_SIZE_K(187)>;
```

Comment:
Actually, I can see that e.g. Matter samples override this to use the entire 256kB (since the flipper core is not used). I think we could do this for this project as well, right?

```
&cpuapp_sram {
	reg = <0x20000000 DT_SIZE_K(256)>;
	ranges = <0x0 0x20000000  0x40000>;
};
```

Reply:
We are doing here an example of an application that still could be modified on the customer site. and it would be confusing why there are problems with booting. So I would leave it as is untill we decide that we need extra space.

Reply:
I can see other samples don't have any issues with increasing this region and I think the customer will be more curious why they don't have all RAM available, but it's OK, we can do it later :).